src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,304 @@
+package duke;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+
+/**
+ * The {@code Duke} class. Main class that drives other functions.
+ */
+public class Duke {
+
+    private final Storage storage;
+    private final TaskList tasks;
```

Comment:
Perhaps naming it listsOfTasks would be more appropriate considering its type TaskList